test/suites/integration/extended_resources_test.go

```diff
@@ -329,6 +409,98 @@ func ExpectNvidiaDevicePluginCreated() {
 	})
 }
 
+// https://github.com/aws-neuron/aws-neuron-sdk/blob/master/src/k8/k8s-neuron-device-plugin.yml
+func ExpectNeuronDevicePluginCreated() {
+	GinkgoHelper()
+	env.ExpectCreated(&appsv1.DaemonSet{
```

Comment:
Seems like the resources are getting not registered? Can we take a look at the DS again?

Reply:
Added missing RBAC permissions for the Neuron device plugin

Reply:
is this thread resolved? Tests seem to be passing.

Reply:
Triggered a new test run to validate